examples/sam_annotator_tracking.py

```diff
@@ -1,21 +1,46 @@
+import os
+from pathlib import Path
+
 from elf.io import open_file
+import pooch
 from micro_sam.sam_annotator import annotator_tracking
 
 
-# This runs the interactive tracking annotator for data from the cell tracking challenge:
-# It uses the training data for the HeLA dataset. You can download the data from
-# http://data.celltrackingchallenge.net/training-datasets/DIC-C2DH-HeLa.zip
 def track_ctc_data():
-    path = "./data/DIC-C2DH-HeLa/train/01"
-    with open_file(path, mode="r") as f:
+    example_data_directory = "./data"
+    with open_file(fetch_example_data(example_data_directory), mode="r") as f:
         timeseries = f["*.tif"][:50]
```

Comment:
Why is the cell tracking timeseries truncated to 50 files? There are 84 tiff files in the training "01" folder.
Should we:
1. Not unzip the full 84 files, but only unzip the 50 files that will be used in this demo?
2. Or, should we remove the `[:50]` from this line, and use the whole series in this demo?

Reply:
Great, done!